src/components/PeopleTable.tsx

```diff
@@ -1,4 +1,28 @@
-export const PeopleTable = () => {
+import React, { useState } from 'react';
+import { Person } from '../types';
+import { SortType, PropName } from './enum';
+
+interface Props {
+  people: Person[],
+  handleSort: (value: string) => void,
+  sortOrder: SortType,
+  propName: string,
+}
+
+export const PeopleTable: React.FC<Props> = ({
+  people,
+  handleSort,
+  sortOrder,
+  propName,
+}) => {
+  const [selectedName, setSelectedName] = useState('');
+  const handleSelection
```

Comment:
It's recommended to wrap the function body with curly braces '{}' for better readability and consistency.